java/src/jmri/jmrix/tmcc/SerialThrottle.java

```diff
@@ -287,8 +287,9 @@ protected void sendToLayout(int value) {
      * @param Mode ignored, as only 32 is valid
      */
     @Override
-    public void setSpeedStepMode(int Mode) {
-        speedStepMode = 32;
+    public void setSpeedStepMode(jmri.SpeedStepMode Mode) {
+        // TODO(austin): figure out what this is supposed to do and fix it.
```

Comment:
TMCC throttles are not NMRA throttles from Lionel. We know we have users using them, so I would leave them alone, even if tests are showing copy-and-paste errors.

Reply:
Ok. This appears to have been unused before and is invalid now, so I've removed it. Thanks for the guidance.